Plots/MapProjections/NCL_maponly_2.py

```diff
@@ -0,0 +1,110 @@
+"""
+NCL_maponly_2.py
+================
+This script illustrates the following concepts:
+    - Outlining continents
+    - Turning on map country boundaries
+    - Changing the color of map outlines
+    - Changing the scale of country boundaries
+    - Zooming in on a particular area
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/maponly_2.ncl
+    - Original NCL plots: https://www.ncl.ucar.edu/Applications/Images/maponly_2_1_lg.png https://www.ncl.ucar.edu/Applications/Images/maponly_2_2_lg.png https://www.ncl.ucar.edu/Applications/Images/maponly_2_3_lg.png
+"""
+
+###############################################################################
+# Import packages
+
+import numpy as np
+import cartopy.crs as ccrs
+import cartopy.feature as cfeature
+import matplotlib.pyplot as plt
+
+from geocat.viz import util as gvutil
+
+###############################################################################
+# Plot:
+
+
+def map_plot(scale, long_min, long_max, lat_min, lat_max, long_labels,
+             lat_labels):
+    '''
+    Args:
+
+        scale : str
+            scale of continent borders and lakes
+        long_min : int
+            minimum longitude for plotting
+        long_max : int
+            maximum longitude for plotting
+        lat_min : int
+            minimum latitude for plotting
+        lat_max : int
+            maximum latitude for plotting
+        long_labels : list
+            list of tick labels for x axis
+        lat_labels : list
+            ist of tick labels for y axis
+
+    Plots a map-only figure with continent borders, country borderes, and lakes
+    at a certain longitude and latitude.
+    '''
```

Comment:
Make sure you're using double quotation marks for docstrings.

Also @michaelavs & @erogluorhan, do we have a standardized way of formatting docstrings for helper functions in geocat-examples?

Reply:
I full agree with @michaelavs .